### ISSUE TITLE
Codegen not respecting alias casing

### ISSUE THREAD

**Issue by Mordil** [apollographql/apollo-ios#1600]
## Bug report

When creating an alias, the generated API does not respect the casing for the alias.

## Versions

Please fill in the versions you're currently using: 

- `apollo-ios` SDK version: 0.34
- Xcode version: 12.2
- Swift version: 5.3
- Package manager: SwiftPM

## Steps to reproduce

using the query

```
query getManifest($input: ManifestInput!) {
  manifest(input: $input) {
    ... on ManifestSuccess {
      legacyBookingID: legacyBookingId
    }
  }
}
```

I get the generated API

```swift
            public static var selections: [GraphQLSelection] {
              return [
                GraphQLField("__typename", type: .nonNull(.scalar(String.self))),
                GraphQLField("legacyBookingId", alias: "legacyBookingID", type: .scalar(String.self)),
              ]
            }

            public var legacyBookingId: String? {
              get {
                return resultMap["legacyBookingID"] as? String
              }
              set {
                resultMap.updateValue(newValue, forKey: "legacyBookingID")
              }
            }
```

**Comment by designatednerd** [comment]
Yeah unfortunately with the typescript codegen this is expected behavior - it uses the JS style of camel-casing even things that are all-caps. Will put this in the "enhancements for new codegen that are actually just leaving stuff alone" pile. 😆

**Comment by calvincestari** [comment]
Hi 👋🏻 - this issue should be resolved in 1.0. 1.0 is currently in alpha releases with a full 1.0 release in H2 2022. Please take a look at the [latest alpha](https://github.com/apollographql/apollo-ios/releases) and let us know if this is still an issue for you.
